property_tests/tests/class_syntax/blob/dfx.json

```diff
@@ -0,0 +1,16 @@
+{
+    "canisters": {
+        "canister": {
+            "type": "azle",
+            "main": "src/index.ts",
+            "candid_gen": "automatic",
+            "declarations": {
+                "output": "test/dfx_generated/canister",
+                "node_compatibility": true
+            },
+            "custom": {
+                "experimental": true
```

Comment:
Experimental must be removed from all class-based examples if possible

Reply:
Done